### ISSUE TITLE
Add usage instructions for train.py to README.md

### ISSUE THREAD

**Issue by ludwig7959** [earthspecies/biodenoising#1]
Any plan to provide training tutorials in README.md?

**Comment by nkundiushuti** [comment]
yeah..that's the plan. this is currently under review. once it's accepted we can add the training instructions.
similarly to demucs denoiser, this assumes taking a csv file with clean (pseudo-clean) and another csv file with noise. these files are generated with generate_training.py . then you need to create your own config file for training
see usage examples in the scripts themselves.

**Comment by JacobGlennAyers** [comment]
I played around with your pretrained model on some data I am interested in and found that it worked very poorly. So, I would be very interested in seeing how effective it is for transfer learning. Let me know when this is up and I can test it on my end as well :)

**Comment by nkundiushuti** [comment]
there are limitations to the method: it will work worse for multiple species vocalizing at the same time, large dynamics (very loud vs very quiet), very noisy recording, species unseen in training. 
I started to work at some point on a domain adaptation branch but I didn't have any time to continue (I have done it up to the point that it does some self-training without ground-truth data and it improved around 1-3dB depending on the dataset but it would be interesting to see if it improves with annotations). also informing it with text or sounds would make it more precise and sort-of source separation tool. if you want to collaborate and start from where I left it, I would be happy to jump in a call

**Comment by JacobGlennAyers** [comment]
Sure, I went ahead and sent you a message on your personal website with my email. Happy to learn more about the project.

**Comment by nkundiushuti** [comment]
hi Jacob, I haven't received the email. could you add me on linkedin?

**Comment by JacobGlennAyers** [comment]
> hi Jacob, I haven't received the email. could you add me on linkedin?

Sure, I just send you a connect request
